packages/es-components/src/components/general-styling-guidelines/GeneralStylingGuidelines.md

```diff
@@ -3,7 +3,7 @@
 A general rule of thumb for setting measurements is as follows:
 
 - If the measurement is focused on text or positioning within or around a
-  container (like padding and margin), use `em` so that if your container's
+  container (like padding and margin), use `rem` so that if your container's
```

Comment:
No. `rem` is based on the global base pixel height. (Usually `16px` unless changed by the user.) `em` is based on the `font-size` of the container. In this case, you want to base your margins/padding on your container.